source/data/user_access.rst

```diff
@@ -17,32 +17,22 @@ Before you can interact with iRODS, as a VSC user you will need to activate the
 
     $ irods-setup | bash
 
-- If you want to connect from any login nodes of other universities' HPC cluster, you should then execute:
+- If you want to connect from any login nodes of other universities' HPC cluster, you should execute:
 
 ::
 
-    $ ssh irods.hpc.kuleuven.be | bash 
+    $ ssh login.hpc.kuleuven.be irods-setup | bash
 
-- If you are connecting from a Tier-2 login node, you should then execute:
-
-::
-
-    $ irods-setup | bash
-
-or,
-
-::
-
-    $ ssh irods.tier1.leuven.vsc | bash
+These commands will activate a temporary token for a period of 7 days. After the 7 days have passed you will need to reactivate your access by re-executing one of these commands again.
 
-**Note**: As you have seen, `irods-setup | bash` command can work both in Tier-1 and Tier-2 login nodes. Besides, to avaoid pseudo-terminal warnings it is recommended to use the same command in compute nodes, meaning in batch scripts once required.
+The Tier-1 Data service has a web accessible endpoint: https://irods.hpc.kuleuven.be/. This allows to browse your collections and the public ones.
```

Comment:
Better not to use the word endpoint to not mix with the Globus endpoints that has a total different meaning

```suggestion
The Tier-1 Data service has a web (GUI) client: https://irods.hpc.kuleuven.be/. This allows to browse your collections and the public ones.
```

Reply:
@jefscheepers @ingridbr I propose to call this "landing page"; platform, nexus or hub are also alternatives? As in itself it's not a client. What was this called initially, as created by Peter? It's not just about the primitive browser included there, but an entrypoint to the different services. "This allows to browse your collections and the public ones."-> "This provides the entrypoint to start working with iCommands, the Python iRODS client, Metalnx and the WebDAV client."

Reply:
It was mostly called 'the Go Portal' internally (because it was written in Go). Personally, I like the sound of 'landing page'. That way, it doesn't sound like it's a place you can do literally everything.